src/python-frontend/function_call_expr.cpp

```diff
@@ -221,15 +250,46 @@ void function_call_expr::handle_chr(nlohmann::json &arg) const
     }
   }
 
+  else if (arg.contains("_type") && arg["_type"] == "Name")
+  {
+    const symbolt *sym = lookup_python_symbol(arg["id"]);
+    if (!sym)
+    {
+      // if symbol not found revert to a variable assignment
+      arg["value"] = std::string(1, static_cast<char>(int_value));
+      typet t = type_handler_.get_typet("chr", 1);
+      exprt expr = converter_.get_expr(arg);
+      expr.type() = t;
+      return expr;
+    }
+    const auto &const_expr = to_constant_expr(sym->value);
+    std::string binary_str = id2string(const_expr.get_value());
+    int_value = std::stoul(binary_str, nullptr, 2);
```

Comment:
`std::stoul()` can `throw std::invalid_argument` or `std::out_of_range`. Shall we handle this exception for malformed binary strings?